low-level/framework/rabbitmq/rabbitmq_egress_accumulated_msgs_processor.py

```diff
@@ -45,28 +48,15 @@ class RabbitMQEgressAccumulatedMsgsProcessor(ScheduledModuleThread, InternalMsgQ
 
     #TODO: read egress config from comman place
     # Section and keys in configuration file
-    RABBITMQPROCESSOR       = 'RABBITMQEGRESSPROCESSOR'
-    VIRT_HOST               = 'virtual_host'
-
-    PRIMARY_RABBITMQ_HOST   = 'primary_rabbitmq_host'
-    EXCHANGE_NAME           = 'exchange_name'
-    QUEUE_NAME              = 'queue_name'
-    ROUTING_KEY             = 'routing_key'
-
-    ACK_QUEUE_NAME          = 'ack_queue_name'
-    ACK_ROUTING_KEY         = 'ack_routing_key'
-
-    USER_NAME               = 'username'
-    PASSWORD                = 'password'
+        # Section and keys in configuration file
+    RABBITMQPROCESSOR       = 'RabbitMQegressProcessor'
     SIGNATURE_USERNAME      = 'message_signature_username'
     SIGNATURE_TOKEN         = 'message_signature_token'
     SIGNATURE_EXPIRES       = 'message_signature_expires'
     IEM_ROUTE_ADDR          = 'iem_route_addr'
-    IEM_ROUTE_EXCHANGE_NAME = 'iem_route_exchange_name'
-
-    SYSTEM_INFORMATION_KEY = 'SYSTEM_INFORMATION'
-    CLUSTER_ID_KEY = 'cluster_id'
-    NODE_ID_KEY = 'node_id'
+    PRODUCER_ID             = 'producer_id'
+    MESSAGE_TYPE            = 'message_type'
```

Comment:
somewhere we need to have reference stating what all the possible values and its meaning

Reply:
nit: one space around the operator.

Reply:
Meaning and possible values can be found from message bus wiki